Packs/APIVoid/ReleaseNotes/1_0_12.md

```diff
@@ -0,0 +1,2 @@
+##### APIVoid
+- Updated README in the integration.
```

Comment:
As the README resides in xsoar.pan.dev and not within XSOAR, we prefer adding this generic ReleaseNotes note usually.
```suggestion
- Documentation and metadata improvements.
```